clients/src/main/java/org/apache/kafka/common/TopicPartitionInfo.java

```diff
@@ -79,9 +95,24 @@ public List<Node> isr() {
         return isr;
     }
 
+    /**
+     * Return the eligible leader replicas of the partition. Note that the ordering of the result is unspecified.
+     */
+    public List<Node> eligibleLeaderReplicas() {
+        return elr;
+    }
+
+    /**
+     * Return the last known eligible leader replicas of the partition. Note that the ordering of the result is unspecified.
+     */
+    public List<Node> lastKnownEligibleLeaderReplicas() {
+        return lastKnownElr;
+    }
```

Comment:
Sorry, @CalvinConfluent, I've changed my mind 🤔 

Using _ELR_ as an established acronym isn't too bad if one has read the KIP. I'd be happy with using `elr()` and `lastKnownElr()` as you originally had it.

I would suggest if you do revert it, just explicitly call it out in the JavaDoc comments.

Thanks!